lib/client/session.go

```diff
@@ -133,6 +133,7 @@ func newSession(ctx context.Context,
 		closeWait:             &sync.WaitGroup{},
 		enableEscapeSequences: enableEscapeSequences,
 		terminal:              term,
+		shouldClearOnExit:     isFIPS() || client.FIPSEnabled,
```

Comment:
nit: Check the pre-computed value first
```suggestion
		shouldClearOnExit:     client.FIPSEnabled || isFIPS(),
```